<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="5" hostname="LIMBO" failures="1" tests="3" name="Test" time="13.64" errors="0" timestamp="2023-12-07T16:20:49 IST">
  <testcase classname="testCases.LoginTestClass" name="loginTest" time="5.687">
    <skipped/>
  </testcase> <!-- loginTest -->
  <testcase classname="testCases.LoginTestClass" name="loginTest" time="3.729">
    <skipped/>
  </testcase> <!-- loginTest -->
  <testcase classname="testCases.LoginTestClass" name="loginTest" time="3.524">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.LoginTestClass.loginTest(LoginTestClass.java:30)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase classname="baseClasses.TestBaseClass" name="@AfterSuite quitDriverSession" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite quitDriverSession -->
  <testcase classname="baseClasses.TestBaseClass" name="@AfterSuite quitDriverSession" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite quitDriverSession -->
  <testcase classname="baseClasses.TestBaseClass" name="@AfterSuite quitDriverSession" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite quitDriverSession -->
  <testcase classname="baseClasses.TestBaseClass" name="@AfterSuite quitDriverSession" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite quitDriverSession -->
  <testcase classname="baseClasses.TestBaseClass" name="@AfterSuite quitDriverSession" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite quitDriverSession -->
</testsuite> <!-- Test -->
